### PR TITLE
Fix broken navigation to application roles tab

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [X] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.roles.v2/components/wizard-updated/role-basics.tsx [hunk 1]
```diff
@@ -233,10 +229,20 @@ export const RoleBasics: FunctionComponent<RoleBasicProps> = (props: RoleBasicPr
      * Navigate to the Applications Edit page.
      */
     const navigateToApplicationEdit = (appId: string) =>
+    // history.push({
+    //     pathname: AppConstants.getPaths().get("APPLICATION_SIGN_IN_METHOD_EDIT")
+    //         .replace(":id", appId).replace(":tabName", `#tab=${ ROLES_TAB_INDEX }`)
+    // });
+
```
Comment (pavinduLakshan) [inline features/admin.roles.v2/components/wizard-updated/role-basics.tsx:236]:
```suggestion
```

**Review by Achintha444 — APPROVED** [review]

File: features/admin.applications.v1/components/edit-application.tsx [hunk 2]
```diff
@@ -518,6 +517,17 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
...
                 }
 
+                return;
+            } else if (urlSearchParams.get(ApplicationManagementConstants.IS_ROLES) === "true") {
+                const rolesTabIndex: number = renderedTabPanes?.findIndex(
+                    (element: {"componentId": string}) =>
+                        element.componentId === ApplicationManagementConstants.ROLES_TAB_URL_FRAG
+                );
+
+                if(rolesTabIndex !== -1) {
```
Comment (JayaShakthi97) [inline features/admin.applications.v1/components/edit-application.tsx:527]:
```suggestion
                if (rolesTabIndex !== -1) {
```

File: features/admin.applications.v1/constants/application-management.ts [hunk 3]
```diff
@@ -110,6 +110,11 @@ export class ApplicationManagementConstants {
      */
     public static readonly IS_PROTOCOL: string = "isProtocol";
 
+    /**
+     * Value for protocol tab navigation.
+     */
+        public static readonly IS_ROLES: string = "isRoles";
```
Comment (JayaShakthi97) [inline features/admin.applications.v1/constants/application-management.ts:116]:
```suggestion
    /**
     * Value for protocol tab navigation.
     */
    public static readonly IS_ROLES: string = "isRoles";
```

Comment (pavinduLakshan) [reply features/admin.applications.v1/constants/application-management.ts]: Fixed with 17bf613

File: features/admin.applications.v1/components/edit-application.tsx (hunk 2 above)
Comment (pavinduLakshan) [reply features/admin.applications.v1/components/edit-application.tsx]: Fixed with 17bf613

**Review by brionmario — APPROVED** [review]

**Pull request merged**